bigchaindb/commands/bigchaindb.py

```diff
@@ -77,11 +66,6 @@ def run_configure(args, skip_if_exists=False):
 
     conf = copy.deepcopy(bigchaindb.config)
 
-    # Patch the default configuration with the new values
-    conf = bigchaindb.config_utils.update(
-            conf,
-            bigchaindb.config_utils.env_config(bigchaindb.config))
-
     print('Generating keypair', file=sys.stderr)
```

Comment:
It doesn't need to generate a keypair. Surely this can be removed.

Maybe just `grep` for "keypair" and nuke all the related code.

Reply:
@ttmc addressed.